connect/README.md

```diff
@@ -50,6 +50,25 @@ const { spawn, message, result } = require("@permaweb/aoconnect");
 
 The duration of this document will use `ESM` for examples
 
+A note about using the package in Next.js. Webpack has an issue where the browser field in the package.json isn't resolved when using the App router. This seems to be related to the fact that Next.js is both a server framework and a browser framework and so the module resolution is possibly tuned to work with this.
```

Comment:
I'd add a header to this section ie. `# Usage with NextJS App Router`.

I'd just link to the outstanding issue on NextJS and the original issue on the ao repo, which contains the analysis.

The snippet below this is good 👍